src/giskardpy/casadi_wrapper.py

```diff
@@ -1996,6 +1996,18 @@ def distance_point_to_line(frame_P_point, frame_P_line_point, frame_V_line_direc
     return distance
 
 
+def distance_point_to_plane(frame_P_current, frame_P_bottom_left, frame_P_bottom_right, frame_P_top_left):
+    ab = frame_P_bottom_right - frame_P_bottom_left
+    ac = frame_P_top_left - frame_P_bottom_left
+
+    normal = cross(ab, ac)
+    d = normal.dot(frame_P_bottom_left)
+    normal.scale(1)
+    normal.scale(d)
```

Comment:
well, there is now point in first scaling it to one and then to d, might as well remove the scale 1